rep-0147.rst

```diff
@@ -0,0 +1,365 @@
+
+Other Interfaces
+================
+
+Battery State
+-------------
+Aerial vehicles should use the ``sensor_msgs/BatteryState`` message for reporting their battery status. [#batterystate]_
+This is a recently standardized message which is much more broadly applicable than aerial vehicles and has already been standardized.
+
+* Datatype: ``sensor_msgs/BatteryState`` [#batterystate]_
+* Topic: ``battery_state``
+
+
+Summary
+=======
+
+Existing interfaces
+-------------------
+
+ - Battery State via ``sensor_msgs/BatteryState`` [#batterystate]_
+ - Trajectory via ``trajectory_msgs/MultiDOFJointTrajectory`` [#multidoftraj]_
+ - Rates via ``mav_msgs/RollPitchYawrateThrust`` [#rpythrust]_ or ``mav_msgs/AttitudeThrust`` [#attitudethrust]_
+ - Accelerations via ``mav_msgs/RateThrust``  [#ratethrust]_
+ - Obstacle information from common ``sensor_msgs`` [#sensormsgs]_
+ - Odometry via``nav_msgs/Odometry`` [#odometry]_
+
+Proposed New Interfaces
+
+ - Extended Odometry via proposed ``nav_msgs/OdometryWithAcceleration`` [propoised]
+
+References
+==========
+
+.. [#astec] astec_mav_framework
+   (http://wiki.ros.org/asctec_mav_framework)
+
+.. [#mavlink2ros] mavlink2ros
+   (https://github.com/posilva/mav2rosgenerator)
+
+.. [#mavros] mavros
+   (http://wiki.ros.org/mavros)
+
+.. [#roscopter] roscopter
+   (https://code.google.com/p/roscopter/)
+
+.. [#crates] CRATES
+   (https://bitbucket.org/asymingt/crates)
+
+.. [#rospilot] rospilot
+   (http://wiki.ros.org/rospilot)
+
+.. [#hector] hector_quadrotor
+   (http://wiki.ros.org/hector_quadrotor)
+
+.. [#autopilot_bridge] autopilot_bridge
+   (https://github.com/mikeclement/autopilot_bridge)
+
+.. [#mav_tools] mav_tools
+   (http://wiki.ros.org/mav_tools)
+
+.. [#djisdk] dji_sdk_dji2ros
+   (http://www.ros.org/browse/details.php?distro=indigo&name=dji_sdk_dji2mav)
+
+.. [#batterystate] sensor_msgs/BatteryState
+   (http://docs.ros.org/api/sensor_msgs/html/msg/BatteryState.html)
+
+.. [#multidoftraj]
+   (http://docs.ros.org/api/trajectory_msgs/html/msg/MultiDOFJointTrajectory.html)
+
+.. [#posestamped]
+   (http://docs.ros.org/api/geometry_msgs/html/msg/PoseStamped.html)
+
+.. [#sensormsgs]
+   (http://wiki.ros.org/sensor_msgs)
+
+.. [#nav_msgs] nav_msgs
+   (http://wiki.ros.org/nav_msgs)
+
+.. [#trajectory_msgs] trajectory_msgs
+   (http://wiki.ros.org/trajectory_msgs)
+
+.. [#dronekit] DroneKit
+   (http://dronekit.io/)
+
+.. [#mav_msgs] mav_msgs
+   (http://wiki.ros.org/mav_msgs)
+
+.. [#mavlink] Mavlink
+   (http://qgroundcontrol.org/mavlink/start)
+
+.. [#mavlink2] Mavlink2
+   (https://github.com/diydrones/mavlink/blob/mavlink2-wip/doc/MAVLink2.md)
+
+.. [#odometry] odometry
+   (http://www.ros.org/doc/api/nav_msgs/html/msg/Odometry.html)
+
+.. [#ratethrust] mav_msgs/RateThrust
+   (http://docs.ros.org/jade/api/mav_msgs/html/msg/RateThrust.html)
+
+.. [#attitudethrust] mav_msgs/AttitudeThrust
+   (http://docs.ros.org/api/mav_msgs/html/msg/AttitudeThrust.html)
```

Comment:
Yeah, I was trying to use the generic url but it defaults to kinetic and not everything is released yet. I'll update those things that aren't in kinetic yet.